app/src/org/commcare/views/widgets/QuestionWidget.java

```diff
@@ -623,10 +639,18 @@ public boolean hasListener() {
         return widgetChangedListener != null;
     }
 
-    public void checkFileSize(File file){
-        if (FileUtil.isFileOversized(file)) {
+    /**
+     * @return True if file is too big to upload.
+     */
+    protected boolean checkFileSize(File file){
+        if (FileUtil.isFileToLargeToUpload(file)) {
+            String fileSize = FileUtil.getFileSize(file) / 1024 + "";
```

Comment:
why are you dividing by 1024 again here, when `getFileSize()` already does that?